specification/monitor/resource-manager/Microsoft.Monitor/preview/2021-06-03-preview/monitoringAccounts_API.json

```diff
@@ -0,0 +1,518 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Monitoring Accounts Control Plane API",
+    "description": "Provides operations for working with Azure Monitoring Accounts",
```

Comment:
looks like this is a new service that is not being onboarded to the RP as a service platform. Can you arrange a meeting with Evan Hissey and Vikash Jain and discuss the reasons why you cannot be on the RpaaS platform? If you already have an exception to not be on the RpaaS platform please send an email to the armapireview alias attaching the exception email.